pkg/cloudprovider/aws/instancetype.go

```diff
@@ -34,6 +35,7 @@ const EC2VMAvailableMemoryFactor = .925
 type InstanceType struct {
 	ec2.InstanceTypeInfo
 	AvailableOfferings []cloudprovider.Offering
+	Options            options.Options
```

Comment:
Can we unwrap this into MaxPods? Passing options seems like it violates cohesion principles.

Reply:
Shall do. I knew it felt bad, but figured I'd get input on how you might want to see it. Shall do.